docs: updated the destination connector docs (#106)
fivetran/fivetran_sdk: development-guide.md

## Destination Connector guidelines

> The destination connector should read the batch files sent by Fivetran and write the data to the destination system. 

Reviewer: ```suggestion
The destination connector should read the batch files sent by Fivetran and write the data to the destination. 
```
I believe we never used "destination system" as a term.

### RPC Calls
#### CreateTable

> The `CreateTable` RPC call should create table, and should fail if it is asked to create a table that already exists. However, it should not fail if the target schema is missing. The destination should create the missing schema.

Reviewer: ```suggestion
The `CreateTable` RPC call should create a table. If you attempt to create a table that already exists, the call will fail. 

If the target schema is missing, the `CreateTable` RPC call should not fail. The destination should create the missing schema.
```
